test/test_site/expected/markbind/js/setup.js

```diff
@@ -24,12 +24,17 @@ function setupAnchorVisibility() {
       jQuery(this).children('.fa.fa-anchor').hide();
     });
   });
+  jQuery('.fa-anchor').each((index, anchor) => {
```

Comment:
Remember to update the content of `test/test_site/expected/markbind/js/setup.js` with the new code that you have.